aliases/available/general.aliases.bash

```diff
@@ -75,9 +80,34 @@ alias rd='rmdir'
 alias shit='bash-it'
 alias batshit='bash-it'
 alias bashit='bash-it'
+alias bsh='bash-it'
+alias batbsh='bash-it'
+alias babsh='bash-it'
 alias bash_it='bash-it'
 alias bash_ti='bash-it'
 
+# Additional bash-it aliases for help/show
+alias bshsa='bash-it show aliases'
+alias bshsc='bash-it show completions'
+alias bshsp='bash-it show plugins'
+alias bshha='bash-it help aliases'
+alias bshhc='bash-it help completions'
+alias bshhp='bash-it help plugins'
+alias bshsch="bash-it search $1"
```

Comment:
Not that I'm anybody who matters, but I usually put my functions in a ~/.bash_functions file.

I think the idea of adding shared functions as a bash-it feature is interesting (enabled/disabled like plugins, etc). Could you put that in as a new feature request rather than discussing it here?